bot/exts/easter/save_the_planet.py

```diff
@@ -0,0 +1,28 @@
+import json
+import random
+from pathlib import Path
+
+from discord import Embed
+from discord.ext import commands
+
+
+class SaveThePlanet(commands.Cog):
+    """A cog that teaches users how they can help our planet."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.embed_data = []
+
+        with open(Path("bot/resources/easter/save_the_planet.json"), 'r', encoding='utf8') as f:
+            self.embed_data = json.load(f)
+
+    @commands.command(aliases=('savetheearth', 'saveplanet', 'saveearth'))
+    async def savetheplanet(self, ctx: commands.Context) -> None:
+        """Responds with a random tip on how to be eco-friendly and help our planet."""
+        return_embed = Embed.from_dict(random.choice(self.embed_data))
```

Comment:
I think we should use our `RandomCycle` utility to make sure these don't repeat in a row.

https://github.com/python-discord/seasonalbot/blob/18500db908281431a1265e5da0948034ee695cbf/bot/utils/randomization.py#L6-L23

We would then open and read the embed data like so:
```py
        with Path("bot/resources/easter/save_the_planet.json").open('r', encoding='utf8') as f:
            EMBED_DATA = RandomCycle(json.load(f))
```
And then this line would look like so:
```suggestion
        return_embed = Embed.from_dict(next(EMBED_DATA))
```

Reply:
I don't think we should do 
```
EMBED_DATA = RandomCycle(json.load(f))
``` 
Unless you can index it by key as well. In the off chance we want to select a specifc embed with a new (non random) command.

Reply:
If that does happen, we can deal with it when it happens. I don't think we need to be preemptive here about this.

Reply:
And I can import `RandomCycle` with this?
```python
from bot.utils import RandomCycle
```

Reply:
No, it is `bot.utils.randomization.RandomCycle`.

Reply:
oh whoops